sdk/python/kfp/_credentials.py

```diff
@@ -12,16 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import abc
 
 from kubernetes.client.configuration import Configuration
 
 
 __all__ = [
     "TokenCredentialsBase",
+    "ML_PIPELINE_SA_TOKEN_ENV",
+    "ML_PIPELINE_SA_TOKEN_PATH",
```

Comment:
Do we really need to expose these two constants?

Reply:
The two constants are interfaces to configure the SDK, so it seems reasonable to me to make them public.

nit: I'd probably suggest a different naming. what do you think about `KF_PIPELINES_SA_TOKEN_ENV` and `KF_PIPELINES_SA_TOKEN_PATH`? to align with https://github.com/kubeflow/pipelines/blob/8b9756247c68b681fc9a7c3ea1ce0dab5e22c847/sdk/python/kfp/_client.py#L84

Reply:
Renaming done

Reply:
I think you forgot to remove these, when the new env vars were added.